ui/src/components/Broker/InviteAcceptScreen.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+
+import { useParty, useLedger } from '@daml/react'
+import { useWellKnownParties } from '@daml/dabl-react'
+import { BrokerInvitation } from '@daml.js/da-marketplace/lib/Marketplace/Broker'
+
+import { wrapDamlTuple } from '../common/damlTypes'
+import { ErrorMessage } from '../common/errorTypes'
+import { InviteAcceptTile, InviteAcceptButton, InviteTextField } from '../common/InviteAcceptTile'
+
+type Props = {
+    onLogout: () => void;
+}
+
+const InviteAcceptScreen: React.FC<Props> = ({ onLogout }) => {
+    const ledger = useLedger();
+    const issuer = useParty();
+    const operator = useWellKnownParties().userAdminParty;
+
+    const [ name, setName ] = useState<string>('')
+    const [ location, setLocation ] = useState<string>('')
+    const [ error, setError ] = useState<ErrorMessage>();
+    const [ loading, setLoading ] = useState(false);
+
+    async function submit() {
+        setLoading(true);
+        const key = wrapDamlTuple([operator, issuer]);
+        const args = { name, location };
+        await ledger.exerciseByKey(BrokerInvitation.BrokerInvitation_Accept, key, args)
```

Comment:
`FormErrorHandled` is used in `InviteAcceptTile.tsx` (which is what is wrapping all of these forms). Let me know if I'm still missing something there.

Reply:
I think I see what you mean - the actual `submit()` functions aren't calling `setError()` here. I think I went with the `console.log()` since we were getting that unknown "could not find template" error that wasn't actually breaking the form, and didn't want to spam the page with it.

Reply:
Hmmm, that error seems to have solved itself, so I'll go ahead and add the `try/catch` blocks on these pages.